go/test/endtoend/tabletmanager/throttler_topo/throttler_test.go

```diff
@@ -306,7 +336,7 @@ func TestThrottlerAfterMetricsCollected(t *testing.T) {
 
 	// By this time metrics will have been collected. We expect no lag, and something like:
 	// {"StatusCode":200,"Value":0.282278,"Threshold":1,"Message":""}
-	//
+	// Note: by WHAT time? We just created a fresh cluster and the throttler is disabled...
```

Comment:
By the time `TestInitialThrottler` is complete and we've moved to `TestThrottlerAfterMetricsCollected`.

In `TestInitialThrottler` we enable the throttler and validate responses... So by this time it's already up and running.

Reply:
Thanks! As we discussed in Slack, somehow I managed to have an incorrect understanding of `TestMain` until today! 🙂